stable/yugabyte/templates/multicluster/service-per-pod.yaml

```
@@ -15,8 +16,15 @@ metadata:
 spec:
   ports:
     {{- range $label, $port := $server.ports }}
+      {{- if (eq $label "grpc-ybc-port")}}
+        {{- if (($root.Values.ybc.enabled))}}
     - name: {{ $label | quote }}
       port: {{ $port }}
+        {{- end}}
+      {{- else}}
+    - name: {{ $label | quote }}
+      port: {{ $port }}
+      {{- end}}
```

```suggestion
    {{- if (eq $label "grpc-ybc-port") }}
    {{- if $.Values.ybc.enabled }}
    - name: {{ $label | quote }}
      port: {{ $port }}
    {{- end }}
    {{- else }}
    - name: {{ $label | quote }}
      port: {{ $port }}
    {{- end }}
```

How about keeping the indentation of template conditions same as YAML block, it is easy to add new YAML blocks that way.

---

I think much difficult to read but fine.